test/java/org/chromium/net/CronetUrlRequestContextTest.java

```diff
@@ -1117,7 +1117,10 @@ public void testInitEngineAndStartRequest() {
   @Test
   @SmallTest
   @Feature({"Cronet"})
-  public void testInitEngineStartTwoRequests() throws Exception {
+  @Ignore(
```

Comment:
@carloseltuerto could you help me take a look at this test failure? It's unclear to me why my changes would cause this to start failing. Is fixing this a blocker to removing the engine singleton?

Reply:
This test is straightforward: a single engine where each request is sent sequentially.

To make sure there is no test pollution, commenting out all the other tests could help. (I don't know how to ask a single java test to run with bazel)

Reply:
You're right, this test does pass if I run it in isolation, disabling all other tests in this file. Thanks for the pointer, I can look into what's not being properly cleaned up.